Sources/Plasma/FeatureLib/pfPython/cyMiscGlue.cpp

```diff
@@ -232,7 +232,7 @@ PYTHON_GLOBAL_METHOD_DEFINITION(PtSendRTChat, args, "Params: fromPlayer,toPlayer
     PyObject* fromPlayerObj = nullptr;
     PyObject* toPlayerListObj = nullptr;
     ST::string message;
-    uint32_t msgFlags = 0;
+    long msgFlags = 0;
     const char* err = "PtSendRTChat expects a ptPlayer, a sequence of ptPlayers, a string, and an optional long";
 
     if (!PyArg_ParseTuple(args, "OOO&|l", &fromPlayerObj, &toPlayerListObj,
```

Comment:
```suggestion
    unsigned int msgFlags = 0;
    const char* err = "PtSendRTChat expects a ptPlayer, a sequence of ptPlayers, a string, and an optional long";

    if (!PyArg_ParseTuple(args, "OOO&|I", &fromPlayerObj, &toPlayerListObj,
```

`cyMisc::SendRTChat()` accepts a `uint32_t` for the flags field, so it would be better to instruct Python to use the correct type. In this case, `unsigned int` is the best match, so update the initial declaration and the format string.

Reply:
I'll apply the suggestion. Lemme validate the change on macOS real quick afterwards.